content/source/docs/cloud/integrations/service-now/index.html.md

```diff
@@ -7,7 +7,7 @@ description: |-
 
 # Terraform ServiceNow Service Catalog Integration Setup Instructions
 
--> **Integration version:**  v1.1.0
+-> **Integration version:**  v2.0.0
 
 -> **Note:** The ServiceNow Catalog integration is designed for use by Terraform Enterprise customers. We do not currently recommend using it with the SaaS version of Terraform Cloud.
```

Comment:
Let's remove this, since we'll be supporting this going forward

Reply:
Changed this as a note to v1 docs